### PR TITLE
Add user-cont/conu for managing updates

### PR DESCRIPTION
**Pull request by fridex**
Depends-On: https://github.com/user-cont/conu/pull/250

### CODE REVIEW

File: config/thoth.yaml [hunk 1]
```diff
@@ -44,3 +44,12 @@ repositories:
     slug: thoth-station/jupyter-notebook
   - <<: *update_config
     slug: thoth-station/sesheta
+  - slug: user-cont/conu
+    token: '{KEBECHET_TOKEN}'
+    managers:
+      - name: update
+        configuration:
+          labels:
```
Comment (TomasTomecek) [inline config/thoth.yaml]: I tried to look up what does this configuration mean, could you please elaborate? Will the bot set these labels on a PR?

Comment (goern) [reply config/thoth.yaml]: ja

Comment (fridex) [reply config/thoth.yaml]: @TomasTomecek if you want to have different labels, just let us know

Comment (TomasTomecek) [reply config/thoth.yaml]: do we need to create them first?

Comment (fridex) [reply config/thoth.yaml]: > do we need to create them first?

I think they are created by github automatically if you request to use them on github API.

Comment (TomasTomecek) [reply config/thoth.yaml]: cool, let us know if github complains

**Pull request merged**